File: wxrtc/src/main/java/com/wx/rtc/bean/SignalCommand.java
package com.wx.rtc.bean;

/**
 * @author wjf
 */
public @interface SignalCommand {

    /*-------------------发送--------------*/
    String START_PROCESS = "start_process";
    String END_PROCESS = "end_process";
    String SEND_ROOM_MSG = "send_room_msg";
    String PROCESS_CMD = "process_cmd";
    String RECORD_CMD = "record_cmd";
    String ENTER_ROOM = "enter_room";
    String EXIT_ROOM = "exit_room";
    String LOGIN = "login";
    String LOGOUT = "logout";
    String CALL_CMD = "call_cmd";


    /*-------------------接受--------------*/
    String ENTER_ROOM_BACK = "enter_room_back";
    String LOGIN_BACK = "login_back";
    String LOGOUT_BACK = "logout_back";
    String START_RECORD_BACK = "start_record_back";
    String END_RECORD_BACK = "end_record_back";
    String GET_UNPUBLISH = "get_unpublish";
    String EXIT_ROOM_BACK = "exit_room_back";
    String REMOTE_ENTER_ROOM = "remote_enter_room";
    String REMOTE_EXIT_ROOM = "remote_exit_room";
    String ROOM_MSG_REV = "room_msg_rev";
    String CALL_MSG_REV = "call_msg_rev";
}

File: wxrtc/src/main/java/com/wx/rtc/bean/SignalCommand.kt
package com.wx.rtc.bean

/**
 * @author wjf
 */
internal annotation class SignalCommand {
    companion object {
        /*-------------------发送--------------*/
        const val START_PROCESS: String = "start_process"
        const val END_PROCESS: String = "end_process"
        const val SEND_ROOM_MSG: String = "send_room_msg"
        const val PROCESS_CMD: String = "process_cmd"
        const val RECORD_CMD: String = "record_cmd"
        const val ENTER_ROOM: String = "enter_room"
        const val EXIT_ROOM: String = "exit_room"
        const val LOGIN: String = "login"
        const val LOGOUT: String = "logout"
        const val CALL_CMD: String = "call_cmd"
        const val VIDEO_AVAILABLE: String = "video_available"
        const val AUDIO_AVAILABLE: String = "audio_available"


        /*-------------------接受--------------*/
        const val ENTER_ROOM_BACK: String = "enter_room_back"
        const val LOGIN_BACK: String = "login_back"
        const val LOGOUT_BACK: String = "logout_back"
        const val START_RECORD_BACK: String = "start_record_back"
        const val END_RECORD_BACK: String = "end_record_back"
        const val GET_UNPUBLISH: String = "get_unpublish"
        const val EXIT_ROOM_BACK: String = "exit_room_back"
        const val REMOTE_ENTER_ROOM: String = "remote_enter_room"
        const val REMOTE_EXIT_ROOM: String = "remote_exit_room"
        const val ROOM_MSG_REV: String = "room_msg_rev"
        const val CALL_MSG_REV: String = "call_msg_rev"
    }
}

